internal/apps/dop/endpoints/iteration.go

```diff
@@ -235,6 +237,33 @@ func (e *Endpoints) DeleteIteration(ctx context.Context, r *http.Request, vars m
 		return errorresp.ErrResp(err)
 	}
 
+	// 检查 iteration 下是否有需求/任务/bug;
+	issueReq := pb.PagingIssueRequest{
+		ProjectID:   iteration.ProjectID,
+		IterationID: int64(iterationID),
+		External:    true,
+		PageNo:      1,
+		PageSize:    10000,
```

Comment:
Is there a parameter to only retrieve the total, as retrieving issue details consumes a lot of resources.

Reply:
No, but your proposal is great. I plan to write a function that iteratively obtains a list of issue IDs